addons/mail/static/tests/helpers/mock_server/models/discuss_channel.js

```diff
@@ -204,18 +204,16 @@ patch(MockServer.prototype, {
     _mockDiscussChannelMessagePost(id, kwargs, context) {
         const message_type = kwargs.message_type || "notification";
         const channel = this.getRecords("discuss.channel", [["id", "=", id]])[0];
-        if (channel.channel_type !== "channel") {
-            const memberOfCurrentUser = this._mockDiscussChannelMember__getAsSudoFromContext(
-                channel.id
-            );
-            if (memberOfCurrentUser) {
-                this.pyEnv["discuss.channel.member"].write([memberOfCurrentUser.id], {
-                    last_interest_dt: serializeDateTime(today()),
-
-                    is_pinned: true,
-                });
+        const members = this.getRecords("discuss.channel.member", [
+            ["channel_id", "=", channel.id],
+        ]);
```

Comment:
Looks like only ids are used + using `pyEnv` should be the preferred way to access mock server data. The following `map` is not required anymore.
```suggestion
        const members = this.pyEnv["discuss.channel.member"].search([
            ["channel_id", "=", channel.id],
        ]);
```